frontend/public/components/dashboard/dashboards-page/dashboards.tsx

```diff
@@ -6,69 +6,20 @@ import { Helmet } from 'react-helmet';
 import { useTranslation } from 'react-i18next';
 
 import { ClusterDashboard } from './cluster-dashboard/cluster-dashboard';
-import { HorizontalNav, PageHeading, LoadingBox, Page, AsyncComponent } from '../../utils';
-import Dashboard from '@console/shared/src/components/dashboard/Dashboard';
```

Comment:
Is OCS the only one using this? Even if so it might be useful for others.

Reply:
1) Yes, only OCS is using `Page & AsyncComponent` from 'utils'. Though `HorizontalNav, PageHeading & LoadingBox` from the `utils` are being used by cluster-dashborad as well and are imported below.
2) `Dashboard` component is also used by both OCS and cluster-dashboard, but for cluster-dashboard this `Dashboard` component is imported in different file named `cluster-dashboard.tsx`.
Please have a look and if you want I can merge  `cluster-dashboard.tsx` and `dashboards.tsx` into one file.

Reply:
got it... u want me to export these functions from here and import them in `ocs-dashboards.tsx` file that I created, right??